core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -2497,6 +2497,35 @@ public void testMinSeqNoBasedAPI() throws IOException {
         }
     }
 
+    public void testGetSnapshotBetweenAPI() throws IOException {
+        final int operations = randomIntBetween(2, 8096);
+        long seqNo = 0;
+        for (int i = 0; i < operations; i++) {
+            translog.add(new Translog.NoOp(seqNo++, 0, "test'"));
+            if (rarely()) {
+                translog.rollGeneration();
+            }
+        }
+        translog.rollGeneration();
+
+        int iters = randomIntBetween(8, 32);
```

Comment:
This can be final. This makes it easier to immediately see what is and is not immutable.